### PR TITLE
Room.Players null check

### PR DESCRIPTION
**Pull request by Thundermaker300**

### CODE REVIEW

File: Exiled.API/Features/Room.cs [hunk 1]
```diff
@@ -76,7 +76,7 @@ public class Room : MonoBehaviour
         /// <summary>
         /// Gets a <see cref="IEnumerable{T}"/> of <see cref="Player"/> in the <see cref="Room"/>.
         /// </summary>
-        public IEnumerable<Player> Players => Player.List.Where(player => player.IsAlive && player.CurrentRoom.Transform == Transform);
+        public IEnumerable<Player> Players => Player.List.Where(player => player.IsAlive && player.CurrentRoom != null && player.CurrentRoom.Transform == Transform);
```
Comment (joker-119) [inline Exiled.API/Features/Room.cs:79]: ``&& (!(player.CurrentRoom is null) && player.CurrentRoom.Transform == Transform)``

Comment (Thundermaker300) [reply Exiled.API/Features/Room.cs:79]: hmmmm yes

**Review by PintTheDragon — CHANGES_REQUESTED** [review]
Do we have access to `is not null`? If so, please change it to that, otherwise feel free to mark this as resolved.

File: Exiled.API/Features/Room.cs [hunk 2]
```diff
@@ -76,7 +76,7 @@ public class Room : MonoBehaviour
         /// <summary>
         /// Gets a <see cref="IEnumerable{T}"/> of <see cref="Player"/> in the <see cref="Room"/>.
         /// </summary>
-        public IEnumerable<Player> Players => Player.List.Where(player => player.IsAlive && player.CurrentRoom.Transform == Transform);
+        public IEnumerable<Player> Players => Player.List.Where(player => player.IsAlive && !(player.CurrentRoom is null) && player.CurrentRoom.Transform == Transform);
```
Comment (NaoUnderscore) [inline Exiled.API/Features/Room.cs:79]: Pattern matching sucks in C# 7.2 or whatever is it, just `!= null`

**Review by NaoUnderscore — CHANGES_REQUESTED** [review]

File: Exiled.API/Features/Room.cs (hunk 2 above)
Comment (joker-119) [reply Exiled.API/Features/Room.cs:79]: or !(is null)

Comment (NaoUnderscore) [reply Exiled.API/Features/Room.cs:79]: `!= null` is just better in this case, cause it doesn't destroys the pattern matching

Comment (joker-119) [reply Exiled.API/Features/Room.cs:79]: Well, I'm bumping the project to C# 9 and re-working all of the null checks to use pattern matching `is not null` or `is null` anyways so, it won't really matter, I'd just be changing this either way.

**Pull request merged**